### DISCUSSION TITLE
Is there an example of a Console app taking in parameters?

### DISCUSSION THREAD

**Discussion by sturlath** [abpframework/abp#7735] [Q&A]
I have used [CommandLineUtils](https://github.com/natemcmaster/CommandLineUtils) to work with parameters. Is there a "best practice" example on how to work with parameters in the Console template?

**Comment by maliming** [comment]
The template for the **Console** application is very simple. There are no best practices yet.

**Reply by sturlath** [reply]
yeah, I figured ;-) I was just wondering if there was an example of how to pipe parameters through it... I just simplified it greatly since it's just a one-off throwaway console at my end. 

But a console app (in my opinion) will always need parameters so hopefully will see that someday in this template.
